acoustid/data/stats.py

```
@@ -89,7 +101,7 @@ def unpack_lookup_stats_key(key: Union[str, bytes]) -> Tuple[str, str, int, str]
     raise ValueError("invalid lookup stats key")
 
 
-def update_lookup_counter(redis, application_id, hit):
+def update_lookup_counter(redis, application_id: int, hit: bool) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `redis`

(ANN001)